mantidproject/mantid
# New loader for LAMP exports

This PR adds a skinny convenience algorithm for loading HDF5 files produced by LAMP (ILL).
It will facilitate the validation of new mantid algorithms against the results produced by the older software LAMP. It will enable hosting the Mantid/LAMP comparisons in Mantid.

**To test:**
Use the algorithm to load the attached file (chop off the .zip extension).
Open it also with e.g. HDFView and make sure the DATA, errors, X and Y axes match with the loaded mantid workspace.
[967076_LAMP.hdf.zip](https://github.com/mantidproject/mantid/files/1575746/967076_LAMP.hdf.zip)




Fixes #21454 . 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
